src/tests/serverless/mockData.json

```diff
@@ -2,7 +2,7 @@
   "headers": {
     "Content-Type": "application/json"
   },
-  "body": "{\"email\": \"*insert the email of the user whose attributes are to be updated.*\"}",
+  "body": "{\"email\": \"test-user@example.fi\"}",
```

Comment:
Lets leave the mock data generic, please revert to have the instructions here rather than the test user email